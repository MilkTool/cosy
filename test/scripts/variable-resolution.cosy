var a = 'global'
{
  fn showA() {
    print a
  }

  showA()
  var a = 'block'
  print a
  showA()
}
