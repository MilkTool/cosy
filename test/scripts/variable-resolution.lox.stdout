[line 8] Error at "a": Shadows existing variable.
